### PR TITLE
{chem}[intel/2018b] ABINIT v8.10.2

### PR DESCRIPTION
**Pull request by jmbeuken**
use libXC 3.0.1

### CODE REVIEW

File: easybuild/easyconfigs/a/ABINIT/ABINIT-8.10.2-intel-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+dependencies = [
+    ('libxc', '3.0.1'),
+    ('netCDF', '4.6.1'),
+    ('netCDF-Fortran', '4.4.4'),
+    ('HDF5', '1.10.2'),
+]
+
+# ensure mpi and intel toolchain
+configopts = '--enable-mpi '
+configopts += '--enable-gw-dpc '
```
Comment (smoors) [inline easybuild/easyconfigs/a/ABINIT/ABINIT-8.10.2-intel-2018b.eb]: @jmbeuken is double precision really needed for GW calculations? note that it is marked 'EXPERIMENTAL'.

File: easybuild/easyconfigs/a/ABINIT/ABINIT-8.10.2-intel-2018b.eb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+configopts += '--with-linalg-flavor=mkl --with-fft-flavor=dfti --with-fft-libs="$LIBFFT" '
```
Comment (smoors) [inline easybuild/easyconfigs/a/ABINIT/ABINIT-8.10.2-intel-2018b.eb]: @jmbeuken is there any performance benefit with `dfti` vs. `fftw3-mkl`?

File: easybuild/easyconfigs/a/ABINIT/ABINIT-8.10.2-intel-2018b.eb (hunk 1 above)
Comment (gmatteo) [reply easybuild/easyconfigs/a/ABINIT/ABINIT-8.10.2-intel-2018b.eb]: The documentation is outdated.  Double precision is ok in GW
Actually, this is the option we use in our testfarm for testing. 
The only drawback is that GW in double precision is slower and more memory demanding...

File: easybuild/easyconfigs/a/ABINIT/ABINIT-8.10.2-intel-2018b.eb (hunk 2 above)
Comment (gmatteo) [reply easybuild/easyconfigs/a/ABINIT/ABINIT-8.10.2-intel-2018b.eb]: These two options are equivalent when the code is linked with MKL as  fftw3-mkl is just wrapping  the native MKL FFT library (DFTI) with an API that is compatible with FFTW3.
fft-flavor DFTI calls the MKL library directly so it's supposed to be slightly faster.

Note, however, that fftw3-mkl should not be used if the threaded FFT algorithm is wanted as  
the wrappers are not thread-safe.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
nic426 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/2b9ecda7d1e1387016729c4b0df2eb11 for a full test report.

File: easybuild/easyconfigs/a/ABINIT/ABINIT-8.10.2-intel-2018b.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/a/ABINIT/ABINIT-8.10.2-intel-2018b.eb]: @jmbeuken can you add a comment that this option is for enabling double precision?

**Comment by smoors** [comment]
Going in, thanks @jmbeuken!

**Pull request merged**